README.md

```diff
@@ -283,6 +283,15 @@ Here are some examples of conditions you can use:
 | `[DOMAIN_EXPIRATION]`      | Resolves into the duration before the domain expires (valid units are "s", "m", "h".)     | `24h`, `48h`, `1234h56m78s`                  |
 | `[DNS_RCODE]`              | Resolves into the DNS status of the response                                              | `NOERROR`                                    |
 
+#### Severity
+
+For failure case you can specify index of severity for each condition. Example of usage: `Low :: [STATUS] == 200`. For case, when severity index is omitted, high severity will be used.
+
+| Available Severity statuses |
```

Comment:
It's just a suggestion, but from my experience usually there are 4 severity types: low, medium, high, and critical

Reply:
Thanks, @skhokhlov! It makes sense! I'll update current PR.

Reply:
@skhokhlov I have updated this PR, please take a look.